### DISCUSSION TITLE
본 매크로로 예약이 쉽지 않다면 다른 매크로를 사용 해보시길 권장드립니다.

### DISCUSSION THREAD

**Discussion by changdoc** [changdoc/naver-vaccine-macro#237] [Announcements]
본 매크로 특성상 설치가 간편하긴하나 동작 구조 상 예약 성공 확률이 낮습니다.

1. 예약 신청페이지에서만 동작 시 -> 새로 고침 주기 짧게 가져가지 못함, 동시에 여러 병원 처리 불가 -> 예약 성공 확률 낮음
2. 네이버 맵(네이버 플레이스)에서 동작 시 -> 잔여 백신 발생시 예약 신청 페이지 이동에 따른 딜레이 존재 -> 예약 신청 자체가 늦을 수 있음 -> 예약 성공 확률이 낮아짐
3. 카카오 맵에서 동작 시 -> 잔여 백신 탐지와 네이버 graphql 데이터 이격이 있는 경우 잔여 백신 발생하더라도 예약 신청 불가 
                                    -> 2. 네이버맵과 마찬가지로 예약 신청 페이지 이동에 따른 딜레이 존재 -> 예약 성공 확률이 낮아짐.

해서 아래 2가지 매크로를 소개해드립니다. 
예상되기로는 예약 신청 처리까지의 딜레이가 본 매크로 보다 빠르다고 판단됩니다.

파이썬으로 브라우저 컨트롤하는 형태의 매크로
https://github.com/SJang1/korea-covid-19-remaining-vaccine-macro

개발자 도구 콘솔 창에서 코드를 넣어 원격 자바스크립트를 바로 실행하는 형태의 매크로
https://github.com/kimytsc/covid-rest-vaccine-macro

**Comment by rladbssam317** [comment]
changdoc님 덕분에 백신 예약에 가까워져서 너무 고맙습니다!
